metrics_utils/src/lib.rs

```diff
@@ -230,6 +242,319 @@ impl BackfillerMetricsConfig {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct Peer2PeerConsistencyMetricsConfig {
+    pub db_bubblegum_get_grand_epochs_latency: Histogram,
+    pub db_bubblegum_get_epochs_latency: Histogram,
+    pub db_bubblegum_get_changes_latency: Histogram,
+    pub db_account_get_grand_buckets_latency: Histogram,
+    pub db_account_get_buckets_latency: Histogram,
+    pub db_account_get_latests_latency: Histogram,
```

Comment:
Why not a single Family<MetricLabel, Histogram>?

Reply:
I usually prefer to have a set of plain metrics, instead of one labeled, because it is much easier to query them by other monitoring systems. But sure, I can turn these into a family. Should I?